dashboard/config/environments/adhoc.rb

```diff
@@ -6,4 +6,27 @@
 Dashboard::Application.configure do
   # Show full error reports
   config.consider_all_requests_local       = true
+
+  # Use digests.
+  config.assets.digest = true
+
+  config.action_controller.perform_caching = true
+  config.serve_static_files = true
+  config.static_cache_control = "public, max-age=86400"
+
+  # Do not fallback to assets pipeline if a precompiled asset is missed.
+  config.assets.compile = false
+
+  # Version of your assets, change this if you want to expire all your assets.
+  config.assets.version = '1.0'
```

Comment:
None, it has never been changed.  If we do change we probably want to refactor so there is a single source of truth.